### DISCUSSION TITLE
Sync screenshots to any cloud image service

### DISCUSSION THREAD

**Discussion by Manila3650** [runelite/runelite#15056] [Ideas]
Sync Screenshot plugin taken pictures to any cloud image service.
RuneLite already has Google integration, why not use syncing to Google Drive? Dropbox? Or any other cloud provider.

**Comment by j03l** [comment]
Even having the ability to sync the photos folder would be awesome.

**Comment by deathbeam** [comment]
Dont think this is doable easily. Just symlink your screenshots folder to your Dropbox folder?

**Comment by deathbeam** [comment]
There is already upload to imgur. If you need to use cloud storage for files, use services made for that, when it is possible with them.

**Comment by XZYZ0R** [comment]
You can sync the directory from the Runelite screenshots to Google Drive by downloading the Google drive software to your pc and connect your acc. You have to do this every time though if you have to reinstall Windows. Ofc it would be easier if this was implemented into Runelite so that it auto syncs to your Google Drive with the profile from Runelite.
